docs/application/web/tutorials/sec-privileges.md

```diff
@@ -3,242 +3,126 @@
+| `http://tizen.org/privilege/mediacontroller.server` | public |  | The application can send information about currently playing media to applications that are allowed to receive it, and can be controlled remotely by those applications. |
+| `http://tizen.org/privilege/messaging.read` | public | Message & Storage | The application can retrieve emails, text messages, and multimedia messages from the server or receive them directly. This may result in additional charges depending on user's payment plan. |
+| `http://tizen.org/privilege/messaging.write` | public | Message & Storage | The application can write text messages, multimedia messages, and emails. This may result in additional charges depending on user's payment plan. |
+| `http://tizen.org/privilege/networkbearerselection` | partner |  | The application can restrict the device so some specific domains can only be accessed via mobile networks. This may result in additional charges depending on user's payment plan. |
+| `http://tizen.org/privilege/nfc.admin` | public |  | The application can change NFC settings, such as turning NFC on or off. |
+| `http://tizen.org/privilege/nfc.cardemulation` | public |  | The application can access smart card details, such as credit card details, and allow users to make payments via NFC. |
+| `http://tizen.org/privilege/nfc.common` | public |  | The application can use NFC common features. |
+| `http://tizen.org/privilege/nfc.p2p` | public |  | The application can push NFC messages to other devices. |
+| `http://tizen.org/privilege/nfc.tag` | public |  | The application can read and write NFC tag information. |
+| `http://tizen.org/privilege/package.info` | public |  | The application can receive package information. |
+| `http://tizen.org/privilege/packagemanager.install` | platform |  | The application can install or uninstall application packages. |
+| `http://tizen.org/privilege/power` | public |  | The application can control power-related settings, such as dimming the screen. |
+| `http://tizen.org/privilege/push` | public |  | The application can receive notifications via the internet. This may result in additional charges depending on user's payment plan. |
+| `http://tizen.org/privilege/recorder` | public | Microphone | The application can record video and audio. |
+| `http://tizen.org/privilege/secureelement` | public |  | The application can access secure smart card chips such as UICC/SIM, embedded secure elements, and secure SD cards. |
+| `http://tizen.org/privilege/setting` | public |  | The application can change and read user settings. |
+| `http://tizen.org/privilege/system` | public |  | The application can read system information. |
+| `http://tizen.org/privilege/systemmanager` | partner |  | The application can read secure system information. |
```

Comment:
Privilege level description is written at the top of the document so the sentence seems not a necessary thing.. but it can be helpful. I don't know why the sentence is added just for the partner level privileges. If it is that important, remark should be added to the platform level privileges' description. Or, I think we can remove all of the sentences remarking the level of the privilege. Need your opinion.

Reply:
This sentence added by https://review.tizen.org/gerrit/gitweb?p=sdk/online-doc.git;a=commit;h=6eb97af711dd812c95e121369de7391999108e66 when I searched the history.

As you said, we have a description on the top of the document, so removing it can be a solution.
If you want to remove the "**Allowed to licensed partners.**", please remove all of them.